src/main/java/com/powsybl/openloadflow/network/LfAction.java

```diff
@@ -46,6 +44,56 @@ public boolean isRelative() {
         }
     }
 
+    public static final class GeneratorUpdates {
+
+        // Reference to the generator
+        private final LfGenerator generator;
+
+        private final Optional<Boolean> relativeActivePower;
```

Comment:
I am not sure it is the best way to do that:
- You can have a new targetP ; I prefer that we store here only the new targetP directly if the "relative" treatment can be done at creation. It should be in per unit (see load action). 
- You can have a new targetQ ; It should be in per unit.
- You can have a new targetV; it should be in per unit. You cannot change the target V of the generator. You have to get the voltage control and change it... but we have an issue ! Because in case of shared control, be need an action that says that we want to change the targetV of all generators of the control... maybe just an exception as a first approach.
- The regulation is more more complex... if the voltage control already exists, you just have to activate the controller. But if not, you have to create the voltage control at network creation and desactivate the controller (that will be activated later...). Maybe leave it to me if you don't need it.